content/videos/challenges/34-diffusion-lmited-aggregation/index.json

```diff
@@ -0,0 +1,46 @@
+{
+    "title": "Diffusion-Limited Aggregation",
+    "description": "In this coding challenge, I explore the generative algorithm \u201cDiffusion-Limited Aggregation\u201d. The pattern is generated from random walkers clustering around a seed (or set of seed) point(s).",
+    "videoNumber": "34",
+    "videoId": "Cl_Gjj80gPE",
+    "date": "2016-08-18",
+    "languages": ["p5.js", "javascript"],
+    "topics": [
+        "Paul Bourke",
+        "Diffusion-Limited Aggregation",
+        "javascript"
+    ],
+    "canContribute": true,
+    "relatedChallenges": [],
+    "timestamps": [],
+    "codeExamples": [
+        {
+            "title": "Diffusion Limited Aggregation",
+            "description": "My visualization of Diffusion Limited Aggregation",
+            "image": "dfa.jpg",
+            "urls": {
+                "p5": "https://editor.p5js.org/codingtrain/sketches/XpS9wGkbB",
+                "processing": "https://github.com/CodingTrain/Coding-Challenges/tree/main/034_DLA/Processing"
+            }
+        }
+    ],
+    "groupLinks": [
+        {
+            "title": "References",
+            "links": [
+                {
+                    "icon": "🔗",
+                    "title": "Diffusion-Limited Aggregation on Wikipedia",
+                    "url": "https://en.wikipedia.org/wiki/Diffusion-limited_aggregation",
+                    "description": "Discussion of DFA."
+                },
+                {
+                    "icon": "📚",
+                    "title": "DLA reference",
+                    "url": "http://paulbourke.net/fractals/dla/",
+                    "description": "Article about DLA."
+                }
```

Comment:
Missing link to [Atom Live Server Package](https://atom.io/packages/atom-live-server)

Reply:
I have added the link to Atom live server.